MANIFEST.in

```diff
@@ -1,7 +1,6 @@
-include *.rst
 include *.txt
 recursive-include cf_units *.py
-include cf_units/etc/site.cfg.template
+include cf_units/etc/_udunits2.pxd
```

Comment:
etc?

Could you simply remove these lines and update the recursive include:

``
recursive-include cf_units *.py *.pxd *.pyx
```